.drone.yml

```diff
@@ -0,0 +1,42 @@
+pipeline:
+  build_4:
+    image: node:4.4.2
+    commands:
+      - npm --loglevel warn install -g npm@3
+      - npm --loglevel warn install
+      - npm run test:ci
+
+  build_5:
+    image: node:5
+    commands:
+      - npm --loglevel warn install
+      - npm run test:ci
+
+  build_app:
+    image: docker:1.11
+    environment:
+      - DOCKER_HOST=tcp://127.0.0.1:2375
+    commands:
+      - docker build -t rotm .
```

Comment:
@easternbloc if we remove the push build, trigger a build on a PR, then make changes to the branch and push - will it rebuild, as technically not a new PR. In these situations we would want a push to trigger a build, else we would have to cancel and resubmit the PR or trigger manually